### PR TITLE
Golint for models/org_team.go

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: models/org_team.go [hunk 1]
```diff
@@ -28,12 +28,12 @@ type Team struct {
...
 func (t *Team) IsOwnerTeam() bool {
-	return t.Name == OWNER_TEAM
+	return t.Name == ownerTeamName
 }
 
-// IsTeamMember returns true if given user is a member of team.
-func (t *Team) IsMember(uid int64) bool {
-	return IsTeamMember(t.OrgID, t.ID, uid)
+// IsMember returns true if given user is a member of team.
+func (t *Team) IsMember(uID int64) bool {
```
[the remark this answers is not in the document]
Comment (tboerger) [reply models/org_team.go]: Or userID

[the remark this answers is not in the document]
Comment (bkcsoft) [reply models/org_team.go]: `userID` please :)

**Comment by thibaultmeyer** [comment]
LGTM

**Comment by strk** [comment]
LGTM

**Comment by thibaultmeyer** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**